src/Traits/LogsActivity.php

```diff
@@ -46,6 +52,16 @@ protected static function bootLogsActivity()
         });
     }
 
+    public function shouldSubmitEmptyLogs (): bool
+    {
+        return !isset(static::$submitEmptyLogs) ? true : static::$submitEmptyLogs;
+    }
+
+    public function isLogEmpty ($attrs): bool
+    {
+        return !count($attrs['attributes'] ?? []) && !count($attrs['old'] ?? []);
```

Comment:
Could you use `empty()` instead of `count()` this method is faster and more clear - it also doesn't use implicit typecasting.

Reply:
Ok, I'll change it.